dinky-admin/src/main/java/org/dinky/configure/AppConfig.java

```diff
@@ -75,16 +73,12 @@ public void addInterceptors(InterceptorRegistry registry) {
         registry.addInterceptor(localeChangeInterceptor());
         // 注册Sa-Token的路由拦截器
         registry.addInterceptor(new SaInterceptor(handler -> {
-                    SaRouter.match("/openapi/**", r -> {
```

Comment:
Don't do this, openapi can't add authentication at the moment, it will cause Dolphinscheduler to be unable to schedule tasks

Reply:
Yes, I realized this, and the last code was only applying 2 new apis